tidy3d/web/config.py

```diff
@@ -3,7 +3,7 @@
 
 from dataclasses import dataclass
 
-SOLVER_VERSION = "release-22.1.3"
+SOLVER_VERSION = "release-22.1.4"
```

Comment:
running all notebooks via `tidy3d-docs/tests/test_notebooks.py` with this new version.  Will inspect and see if everything passes as expected.

Reply:
VizData.ipynb failed because the max monitor memory size limit was violated.  It turned out it was because a FIeldTimeMonitor had an `interval=30` but this wasn't taken into account in the monitor size computation function.  I made fixes and pushed a commit to this branch.